ambari-server/src/main/resources/stack-hooks/before-ANY/scripts/params.py

```diff
@@ -187,13 +187,13 @@ def is_secure_port(port):
 
 user_group = get_cluster_setting_value('user_group')
 
-ganglia_server_hosts = execution_command._execution_command.__get_value("clusterHostInfo/ganglia_server_hosts", [])
-namenode_host = execution_command._execution_command.__get_value("clusterHostInfo/namenode_hosts", [])
-hbase_master_hosts = execution_command._execution_command.__get_value("clusterHostInfo/hbase_master_hosts", [])
-oozie_servers = execution_command._execution_command.__get_value("clusterHostInfo/oozie_server", [])
-falcon_server_hosts = execution_command._execution_command.__get_value("clusterHostInfo/falcon_server_hosts", [])
-ranger_admin_hosts = execution_command._execution_command.__get_value("clusterHostInfo/ranger_admin_hosts", [])
-zeppelin_master_hosts = execution_command._execution_command.__get_value("clusterHostInfo/zeppelin_master_hosts", [])
+ganglia_server_hosts = execution_command.get_value("clusterHostInfo/ganglia_server_hosts", [])
```

Comment:
lets just add a method to execution_command as following to handle this. 

def get_component_hosts(component_name):